src/test/java/edu/harvard/iq/dataverse/api/InfoIT.java

```diff
@@ -82,6 +81,25 @@ public void testGetZipDownloadLimit() {
                 .body("data", notNullValue());
     }
 
+    @Test
+    public void testOpenApiDefinition(){
+        Response response = given()
+                .get("/api/info/openapi/jasonInvalid");
+        response.prettyPrint();
+        response.then().assertThat().statusCode(BAD_REQUEST.getStatusCode());
+ 
+        String jsonDefinitionFileContent = UtilIT.getDatasetJson("src/main/resources/edu/harvard/iq/dataverse/openapi/dataverse_openapi.json");
+        response = given().get("/api/info/openapi/json");
+        response.then().assertThat().statusCode(OK.getStatusCode())
+                .body(equalTo(jsonDefinitionFileContent));
+       
+        String yamlFileContent = UtilIT.getDatasetJson("src/main/resources/edu/harvard/iq/dataverse/openapi/dataverse_openapi.yaml");
+        response = given().get("/api/info/openapi/yaml");
+        response.then().assertThat().statusCode(OK.getStatusCode())
+                .body(equalTo(yamlFileContent));
```

Comment:
So these files are checked into git now in 2024 but will never be updated in git? They'll be frozen in time? Is it a good idea to have them in git?

On disk the files will be updated so these tests are asserting that the files are updated on disk?

I might be confused about what's going on. 😅